src/sage/graphs/graph.py

```diff
@@ -7181,20 +7183,131 @@ def cores(self, k=None, with_labels=False):
             return core
         return list(core.values())
 
-    @doc_index("Leftovers")
-    def modular_decomposition(self, algorithm=None, style='tuple'):
+    @doc_index("Modules")
+    def is_module(self, vertices):
+        r"""
+        Return whether ``vertices`` is a module of ``self``.
```

Comment:
The form `Check whether...` is more common in the graph module.

Reply:
done in 151d10f61fe